test/python/hermitian_density_matrix.py

```diff
@@ -0,0 +1,62 @@
+import triqs.utility.mpi as mpi
+from h5 import HDFArchive
+from triqs.operators import *
+#from atom_diag import trace_rho_op
+from triqs.atom_diag import trace_rho_op
+from triqs_cthyb import *
+from triqs.gf import *
+import numpy as np
+
+def is_hermitian(matrix):
+    N, M = matrix.shape
+    return any([matrix[i,j] == matrix[j,i].conjugate() for i in range(N) for j in range(M)])
```

Comment:
This should be `all` not `any`?
Also you could do `(matrix == matrix.conj().T).all()`